src/article_metrics/management/commands/import_metrics.py

```diff
@@ -78,8 +101,8 @@ def handle(self, *args, **options):
 
         except BaseException:
             msg = "unhandled exception calling the import-metrics command."
```

Comment:
also interesting. search logs for this

Reply:
not much actually. one error back in march, reading from a closed connection or something